IntegrationTests/Assets/PurchasesAPITests.cs

```diff
@@ -0,0 +1,197 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class CustomListener : Purchases.UpdatedCustomerInfoListener
+{
+    public override void CustomerInfoReceived(Purchases.CustomerInfo customerInfo)
+    {
+        throw new System.NotImplementedException();
+    }
+}
+
+public class PurchasesAPITests : MonoBehaviour
+{
+    private void Start()
+    {
+        // TODO: we still have a public method here called GetEntitlments, but it looks like that's 
+        // super old and should be removed
+        Purchases purchases = GetComponent<Purchases>();
+        purchases.deprecatedLegacyRevenueCatAPIKey = "abc";
+        purchases.revenueCatAPIKeyApple = "def";
+        purchases.revenueCatAPIKeyGoogle = "ghi";
+        purchases.appUserID = "abc";
+        purchases.productIdentifiers = new[]
+        {
+            "a", "b", "c"
+        };
+
+        purchases.listener = new CustomListener();
+        purchases.observerMode = true;
+        purchases.userDefaultsSuiteName = "suitename";
+        purchases.proxyURL = "https://proxy-url.revenuecat.com";
```

Comment:
There are some lowercase properties here too

Reply:
another great catch, also updated in #112